Makefile

```diff
@@ -46,7 +46,7 @@ cyclo:
 	@${GOPATH}/bin/gocyclo -over 100 pkg
 
 deadcode:
-	@${GOPATH}/bin/deadcode
+	@deadcode $(go list ./...)
```

Comment:
We should avoid vendor directory here.. this might give false positives for vendor-ed libraries which we cannot fix.  
```
@deadcode $(go list ./... | grep -v vendor |grep -v browser)
```

Reply:
`go list ./...` does not return `vendor` directory.  `browser` does not fail.  Unable to understand your comment.